tsconfig.base.json

```diff
@@ -3,6 +3,7 @@
     "composite": true,
     "declaration": true,
     "declarationMap": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": "."
```

Comment:
hmm, why did we add this back? Was this me? I think we don't need it.

Reply:
We added it because I was getting a build error when importing utils into the davinci-client package
```
nx run @forgerock/davinci-client:build
      Compiling TypeScript files for project "@forgerock/davinci-client"...
      error TS5090: Non-relative paths are not allowed when 'baseUrl' is not set. Did you forget a leading './'?
```

Reply:
ah ok fair enough forgot that.